src/GUI/AppGUI.py

```diff
@@ -68,6 +68,7 @@ class SettingsKey(Enum):
     CPUThresholdTemp = "app/fan/cpu/threshold_temp"
     GPUFanSpeed = "app/fan/gpu/speed"
     GPUThresholdTemp = "app/fan/gpu/threshold_temp"
+    exitValue = "app/exit"
```

Comment:
Capitalize and rename to `MinimizeOnCloseFlag` (or choose other meaningful name)